packages/teleport/package.json

```diff
@@ -25,6 +25,7 @@
   "devDependencies": {
     "@gravitational/build": "^1.0.0",
     "@types/wicg-native-file-system": "^2020.6.0",
-    "jest-canvas-mock": "^2.3.1"
+    "jest-canvas-mock": "^2.3.1",
+    "ts-loader": "^9.3.1"
```

Comment:
Hmm, that's suspicious, the addition of ts-loader to `packages/build/package.json` should've solved the issue. 🤔